p_sets/2/encryption.rb

```diff
@@ -0,0 +1,43 @@
+module Encryption
+
+  ASCII_LETTERS = {
+    capitals: (65..90),
+    lowercase: (97..122)
+  }
+
+  def cyphertext_reporter(plaintext, integer)
+    puts format('%s', encrypt(plaintext, integer))
+  end
+
+  def encrypt(plaintext, string)
+    key = make_key(string)
+    encrypted_ascii = make_encrypted_ascii(plaintext, key)
+    make_cyphertext(encrypted_ascii)
+  end
+
+  def apply_key(character, key)
+    capital_letter?(character) ? size = :capitals : size = :lowercase
+    ascii = character.ord + key
+    ascii > ASCII_LETTERS[size].max ? ascii - 26 : ascii
+  end
+
+  def make_cyphertext(ascii)
+    cyphertext = []
+    ascii.each do |integer|
+      cyphertext << integer.chr
+    end
```

Comment:
Another case for a map. In fact, ruby makes it v easy to map to a function of each element: you can do `ascii.map(&:chr).join` for the whole function.